### PR TITLE
mod: bump lnd to v0.16.0-beta final

### PR DESCRIPTION
**Pull request by guggero**
#### Pull Request Checklist
- [ ] LndServices minimum version has been updated if new lnd apis/fields are
  used.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -18,10 +18,10 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.5.0
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lightninglabs/aperture v0.1.18-beta
-	github.com/lightninglabs/lndclient v0.16.0-9
+	github.com/lightninglabs/lndclient v0.16.0-10
```
Comment (bhandras) [inline go.mod:21]: Do we want aperture to 0.1.19 too?

Comment (guggero) [reply go.mod:21]: Ah, good point, I think I'll tag a `0.1.20-beta` of aperture after bumping the version there.

**Review by ellemouton — APPROVED** [review]

**Review by bhandras — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (bhandras) [reply go.mod:21]: Oh and i think we need that grpc replace too!

**Comment by guggero** [comment]
Bumped aperture version as well.

**Pull request merged**